### PR TITLE
🌐 Add Turkish translation for `docs/tr/docs/advanced/index.md`

### PR DESCRIPTION
**Pull request by hasansezertasan**
🌐 Add Turkish translation for `docs/tr/docs/advanced/index.md`

[Original File](https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/advanced/index.md)

Discussion: #9193

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit ac8d5dd0519b629cf8559c4d4e6512cc7a5cf6b2 at: https://0e5ebbfc.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 8bd4dd11d2916a50e3b309d7ef484081332b2f1f at: https://524782fe.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 031a1f26972e5ecea0deceb8c0f5dc0ebc7dc4b5 at: https://457c05c3.fastapitiangolo.pages.dev

**Review by OzgunCaglarArslan — CHANGES_REQUESTED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit d4972d532e686c6855b8ac37a0f54738d67f11ef at: https://9a158ca2.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 33d29056b1e07fabc7fcdfab5d398f79f2bfcc72 at: https://b5c5b06a.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit b48d429dcb0538dd8a36bb3f175997de4d142316 at: https://1b73cdb9.fastapitiangolo.pages.dev

File: docs/tr/docs/advanced/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Gelişmiş Kullanıcı Rehberi
+
+## Ek Özellikler
+
+[Tutorial - User Guide](../tutorial/index.md){.internal-link target=_blank} sayfası **FastAPI**'ın tüm ana özelliklerini tanıtmaya yetecektir.
+
+İlerleyen bölümlerde diğer seçenekler, konfigürasyonlar ve ek özellikleri göreceğiz.
+
+!!! tip "İpucu"
```
Comment (OzgunCaglarArslan) [inline docs/tr/docs/advanced/index.md:9]:
```suggestion
!!! İpucu
```

File: docs/tr/docs/advanced/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Önce Öğreticiyi Okuyun
+
+[Tutorial - User Guide](../tutorial/index.md){.internal-link target=_blank} sayfasındaki bilgilerle **FastAPI**'nın çoğu özelliğini kullanabilirsiniz.
+
+Sonraki bölümler bu sayfayı okuduğunuzu ve bu ana fikirleri bildiğinizi varsayarak hazırlanmıştır.
+
+## Diğer Kurslar
+
+[Tutorial - User Guide](../tutorial/index.md){.internal-link target=_blank} sayfasının ve bu **Gelişmiş Kullanıcı Rehberi**'nin öğretici olmak amacıyla (bir kitap gibi) yazılmış olmasına ve **FastAPI'ı öğrenmek** için yeterli olmasına rağmen, ek kurslarla desteklemek isteyebilirsiniz.
```
Comment (OzgunCaglarArslan) [inline docs/tr/docs/advanced/index.md:22]:
```suggestion
[Tutorial - User Guide](../tutorial/index.md){.internal-link target=_blank} sayfası ve bu **Gelişmiş Kullanıcı Rehberi**, öğretici bir kılavuz (bir kitap gibi) şeklinde yazılmıştır ve **FastAPI'ı öğrenmek** için yeterli olsa da, ek kurslarla desteklemek isteyebilirsiniz.
```

**Review by OzgunCaglarArslan — CHANGES_REQUESTED** [review]

File: docs/tr/docs/advanced/index.md (hunk 1 above)
Comment (hasansezertasan) [reply docs/tr/docs/advanced/index.md:9]: Admonition notasyonu bu şekildedir, önerini uygularsak admonition bozulur.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 87441468f3288bb871975edc478b8deaf2aa4618 at: https://223123a5.fastapitiangolo.pages.dev

**Review by OzgunCaglarArslan — APPROVED** [review]

**Comment by alejsdev** [comment]
Thank you! @hasansezertasan @OzgunCaglarArslan :partying_face:

**Pull request merged**